src/components/Article.js

```diff
@@ -59,6 +59,9 @@ const ArticleSection = ({
     section: ({ text, anchor }) => {
       // @todo styling to be confirmed with design
       confirmDialog({ message: i18n.i18n('confirm-section', text), onSubmit: () => goToSubpage({ title: anchor }) })
+    },
+    gallery: ({ fileName }) => {
```

Comment:
In other similar link handlers, the key is named after what the target is rather than what will happen when you select it. Following the same style, the key here could be named `image`.

Reply:
true, and make sense, gonna change it!!!